src/libraries/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj

```diff
@@ -261,5 +261,6 @@
     <Reference Include="System.Threading" />
     <Reference Include="System.Threading.Thread" />
     <Reference Include="System.Xml.XDocument" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Text.RegularExpressions/gen/System.Text.RegularExpressions.Generator.csproj"  OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
```

Comment:
Why is this needed?

Reply:
See

https://github.com/dotnet/runtime/blob/cc2cb040cf3c6464269ca3b563ae4cfe705d45f7/eng/generators.targets#L9-L30

for how that works.

Reply:
Yeah, instead of having individual ProjectReferences on each project that uses it, we should instead update generators.targets to have all projects be able to consume the source generator.

Reply:
Ah, thanks.
I originally did it as a project reference because I ran across the dll one in a PR, before I saw the recommendations.  But yes, this would be much better.

Reply:
I've changed to adding the targets to that file, and it works, but I'm not very experienced with some of the more arcane things in build files.  I know this is going to be reviewed anyways, but please be thorough.